src/containers/home/SubscribeForm.js

```diff
@@ -48,21 +48,26 @@ class SubscribeForm extends React.Component {
                 {submitSucceeded && <SuccessPanel header="Successful subscription" onDismiss={() => reset()} />}
                 {!submitSucceeded && (
                     <Form error={error ? true : false} onSubmit={handleSubmit(this.handleSubmit)}>
-                        <ErrorPanel
-                            content={error}
-                            header={"Subscription failed"}
-                            onDismiss={() => {
-                                clearSubmitErrors();
-                            }}
-                        />
+                        {error && (
+                            <p>
+                                aaa
```

Comment:
`bbb`?

Reply:
ccc!
(this file is removed in an other PR so I leave this as is)